.vscode/settings.json

```diff
@@ -1,6 +1,25 @@
 {
-	"editor.defaultFormatter": "biomejs.biome",
-	"[prisma]": {
-		"editor.defaultFormatter": "Prisma.prisma"
-	}
-}
\ No newline at end of file
+  "[prisma]": {
+    "editor.defaultFormatter": "Prisma.prisma"
+  },
+  "cSpell.words": [
+    "clsx",
+    "compa",
+    "iconify",
+    "isbot",
+    "julr",
+    "katex",
+    "lucide",
+    "posthog",
+    "prefs",
+    "rehype",
+    "shikijs",
+    "typecheck",
+    "unocss"
+  ],
+  "editor.formatOnSave": true,
+  "typescript.updateImportsOnFileMove.enabled": "always",
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": "explicit"
+  }
```

Comment:
We don't need these

Reply:
I format files on file save. If it's the standard, any file I touch won't change. Else every file I just check out will get formatted, hence the extra space removals